server/sonar-server/src/main/java/org/sonar/server/ui/ws/GlobalAction.java

```diff
@@ -63,16 +64,18 @@
   private final DbClient dbClient;
   private final OrganizationFlags organizationFlags;
   private final DefaultOrganizationProvider defaultOrganizationProvider;
+  private final UserSession userSession;
 
   public GlobalAction(PageRepository pageRepository, Settings settings, ResourceTypes resourceTypes, Server server,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)